welcome/glossary.adoc

```diff
@@ -0,0 +1,252 @@
+:_mod-docs-content-type: REFERENCE
+[id="getting-started-openshift-common-terms_{context}"]
+= Glossary of common terms for {product-title}
```

Comment:
REVIEWERS: Though this looks like a new file, the content has been just moved from Arch and Getting Started books.